### PR TITLE
Fix codesmells in UT proj

### PR DESCRIPTION
**Pull request by csaba-sagi-sonarsource**

### CODE REVIEW

File: analyzers/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs [hunk 1]
```diff
@@ -979,11 +985,11 @@ public void Cfg_For()
             var cfg = Build("for (var i = 0; true; i++) { var x = 10; }");
             VerifyForStatement(cfg);
             VerifyAllInstructions(cfg.EntryBlock, "0", "i = 0");
-            var condition = cfg.EntryBlock.SuccessorBlocks.First();
+            var condition = cfg.EntryBlock.SuccessorBlocks[0];
```
Comment (pavel-mikula-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs:988]: Why this change? It's not a code smell and `.First()` is easier to understand than the `[0]`

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
One question for now

File: analyzers/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs (hunk 1 above)
Comment (csaba-sagi-sonarsource) [reply analyzers/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs:988]: It is a fix for this rolsyn based info level code smell: https://sonarcloud.io/project/issues?id=sonaranalyzer-dotnet&issues=AXi2n7WIzHMUitPsOt8i&open=AXi2n7WIzHMUitPsOt8i
